plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreIntrospection.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.metastore.thrift;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ThriftHiveMetastoreIntrospection
+{
+    public static final int UNDEFINED_ALTERNATIVE = Integer.MAX_VALUE;
+
+    private final ThriftMetastoreStats stats = new ThriftMetastoreStats();
```

Comment:
Not sure why you moved ThriftMetastoreStats here
that wasn't my intention, sorry if i told you wrong

my intention was to move `private final MetastoreSupportsDateStatistics metastoreSupportsDateStatistics = new MetastoreSupportsDateStatistics();` here.

Reply:
My bad, I mixed the names and blindly followed